de.metas.business.rest-api/src/main/java/de/metas/rest_api/payment/JsonPaymentInfo.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * #%L
+ * de.metas.business.rest-api
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.rest_api.payment;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import de.metas.rest_api.common.JsonExternalId;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+import java.math.BigDecimal;
+
+@Value
+@Builder
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY, getterVisibility = JsonAutoDetect.Visibility.NONE, isGetterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class JsonPaymentInfo
+{
+	@ApiModelProperty(required = true, //
+			dataType = "java.lang.String", //
+			value = "This translates to `C_BPartner.ExternalId`.")
+
+	@NonNull
+	JsonExternalId externalBpartnerId;
```

Comment:
Please instead have "String bpartnerIdentifier" here.
See `de.metas.rest_api.utils.IdentifierString` and its uses.
See e.g. `de.metas.rest_api.bpartner_pricelist.BpartnerPriceListRestController.getSalesPriceList(String, String, String)` (note that there even is already a canned swagger-doc for this)